public/index.html

```diff
@@ -0,0 +1,741 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Brand Marketing App</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Project Week One">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300,100,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700,300' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway:400,100,300,500,600' rel='stylesheet' type='text/css'>
+    <link rel='stylesheet' href='css/stylesheet.css' type='text/css'>
+  </head>
+  <body>
+    <section class="home" id="home">
+      <section class="a1-home-header">
+        <section class="appbox-logo-section">
+          <img class= "appbox-logo" src="./img/appbox.png" alt="appbox logo">
+        </section>
+        <nav class="home-nav">
+          <ul>
+            <li><a class="home-nav-a" href="#home">HOME</a></li>
+            <li><a class="home-nav-a" href="#about">ABOUT</a></li>
+            <li><a class="home-nav-a" href="#features">FEATURES</a></li>
+            <li><a class="home-nav-a" href="#reviews">REVIEWS</a></li>
+            <li><a class="home-nav-a" href="#screens">SCREENS</a></li>
+            <li><a class="home-nav-a" href="#blog">BLOG</a></li>
+            <li><a class="home-nav-a" href="#support">SUPPORT</a></li>
+            <li class="icon-seek"></li>
```

Comment:
Try not to use empty elements purely for styling. Additionally, given that you're using a search icon here, this really should be something that's actionable (either opening up a search box or taking a user to a page where they can perform a search). At the very least, this should have an  `a` that has the icon attached to it.